components/scream/tests/rrtmgp/input.yaml

```diff
@@ -14,6 +14,11 @@ Grids Manager:
   Type: Physics Only
   Reference Grid: Physics
   Physics Only:
-    Number of global columns:   10
-    Number of vertical levels: 128
+    Number of global columns: 384
+    Number of vertical levels: 72
+
+# The name of the file containing the initial conditions for this test.
+# TODO: Actually create an input file for rrtmgp.  Note that currently, rrtmgp is initialized using YAKL and not by reading input.
```

Comment:
Just commenting that I need to fix this comment.  We now have an input file for rrtmgp, but it may still not get used because of YAKL.  Will need to check when the RAD PR goes in.